### PR TITLE
Fixes Karma Firefox test inconsistency

### PR DESCRIPTION
**Pull request by evertonfraga**
This PR will be a sandbox of some different approaches to make FF + Karma work on our CI properly.

Measures:
- [x] Disable other workflows for now, so we can make more tests at once, without further impact on CI
- [x] Isolate FF run from Chrome, to discard timeout due to intensive resource usage

Hypothesis:
- [x] Run Chrome and Firefox in sequence, rather than in parallel

Cleanup:
- [x] Re-enable other workflow files
- [x] Revert Block workflow file to initial state.

Note: The commits will be squashed at the end, so the history will be clean.

### CODE REVIEW

File: packages/block/karma.conf.js [hunk 1]
```diff
@@ -6,8 +6,9 @@ module.exports = function(config) {
     preprocessors: {
       './test-build/**/*.js': ['browserify'],
     },
+    concurrency: 1,
```
Comment (evertonfraga) [inline packages/block/karma.conf.js:9]: The simplest solution to an annoying problem :)

Comment (holgerd77) [reply packages/block/karma.conf.js:9]: Can you update this to a merge-ready PR? 😄

Comment (evertonfraga) [reply packages/block/karma.conf.js:9]: Sure! doing it now.

Was waiting for the approval of you all, so I can clean the git tree and push it forward.

File: packages/block/karma.conf.js [hunk 2]
```diff
@@ -6,8 +6,9 @@ module.exports = function(config) {
...
     reporters: ['dots'],
-    browsers: ['ChromeHeadless'],
+    browsers: ['FirefoxHeadless', 'ChromeHeadless'],
```
Comment (evertonfraga) [inline packages/block/karma.conf.js:11]: Adding firefox back

**Review by ryanio — APPROVED** [review]
LGTM, thanks for debugging this, it was stumping me!

**Comment by holgerd77** [comment]
Nice. 😄

**Pull request merged**